CraftTweaker2-API/src/main/java/crafttweaker/api/tooltip/IngredientTooltips.java

```diff
@@ -169,14 +178,16 @@ public String describe() {
     private static class ClearTooltipAction implements IAction {
         
         private final IIngredient ingredient;
-        
-        public ClearTooltipAction(IIngredient ingredient) {
+        private final boolean leaveName;
+
+        public ClearTooltipAction(IIngredient ingredient, boolean leaveItemName) {
```

Comment:
Also a breaking change, maybe leave the original constructor in with a `false` as the default?